### PR TITLE
Feature hello

### PR DESCRIPTION
**Pull request by Obote**
### What does this PR do?
- Setup git workflows
- Create hello-microverse app
- Update README.md

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (ZawadiSumba66) [inline .github/workflows/linters.yml:1]: - [ ] One linter check is failing, the `stlylelint` check. All checks should be green inorder to achieve an error free code.Please fix this

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (ZawadiSumba66) [inline .gitignore]: - [ ] The `.gitignore` file should also contain a `test.md` file that has to be created in your local environment but ignored by this file.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi  @Obote 👋 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Please ensure that you personalize your README file content. This makes your repo look more professional. 
- [ ] Also ensure that you make your pull request more descriptive. A pull request should have a short summary describing introduced changes in more detail.

_Check for other requested changes in the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Approved 👏 

Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**